go/vt/vtgate/safe_session.go

```diff
@@ -370,6 +371,22 @@ func (session *SafeSession) SetLockSession(lockSession *vtgatepb.Session_ShardSe
 	session.mu.Lock()
 	defer session.mu.Unlock()
 	session.LockSession = lockSession
+	session.LastLockHeartbeat = time.Now().Unix()
+}
+
+//UpdateLockHeartbeat updates the LastLockHeartbeat time
```

Comment:
nit: our coding standards require a space between `//` and the first word :).

Reply:
We should teach the linter so we don't forget